config/routes.rb

```diff
@@ -4,4 +4,6 @@
   get      '/login', to: 'sessions#new', as: "login"
   get      '/signup', to: 'users#new'
   post     '/login', to: 'sessions#create'
+  post     '/signup', to: 'users#create'
```

Comment:
I think to be restful we need to change this '/signup' to '/users'? Does the client have an opinion? @s-espinosa @AliSchlereth